docs/advanced_topics/customization/page_editing_interface.md

```diff
@@ -292,3 +292,176 @@ class BlogPage(Page):
+
+It's best to add this to a `wagtail_hooks.py` file as this will get run at the right time.
+
+```py
+# wagtail_hooks.py
+
+from django.db import models
+from wagtail.admin.forms.models import register_form_field_override
+from wagtail.admin import widgets
+
+# .. other imports & hooks
+
+
+register_form_field_override(
+    models.SlugField,
+    override={"widget": widgets.SlugInput(locale="uk-UK")},
+)
+```
+
+The following sections will only focus on the `SlugInput(...` usage and not where to override this.
+
+### Overriding the default locale behavior via `locale`
+
+The `SlugInput` is locale aware and will adjust the transliteration (Unicode to ASCII conversion) based on the most suitable locale, only when [`ALLOW_UNICODE_SLUGS`](wagtail_allow_unicode_slugs) is `False`.
+
+The locale will be determined from the target translation locale if ()[internationalisation] is enabled.
+
+If Internationalization is not in use, it will be based on the language of the admin for the current logged in user. This can be overridden, see below.
+
+#### Examples
+
+| `SlugInput`                                             | Description                                                                                                                                          |
+| ------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `SlugInput(locale="uk-UK")` or `SlugInput(locale="uk")` | Override the locale for a specific language code only (Ukrainian in this example)                                                                    |
+| `SlugInput(locale=False)`                               | Avoid the default logic for determining the locale and do not attempt to be locale aware, transliteration will still apply but use a basic approach. |
+| `SlugInput(locale=my_page_instance.locale)`             | Use a [`Locale`](locale_model_ref) instance from some other model source.                                                                            |
+
+### Adding custom formatters via `formatters`
+
+`SlugInput` also accepts a `formatters` kwarg, allowing a list of custom formatters to be applied, each formatter item can be one of the following.
+
+1. A regex pattern or string (for example `r"\d"`, `re.compile(r"\d")`)
+2. A list-like value containing a regex pattern/string and a replacement string (for example `[r"\d", "n"]`)
+3. A list-like value containing a regex pattern/string and a replacement string & custom base JavaScript regex flags (for example `[r"\d", "n", "u"]`)
+
+A reminder to ensure that regex strings are appropriately escaped.
+
+As an example below, here's a formatter that will remove common stop words from the slug. If the title is entered as `The weather and the times`, this will produce a slug of `weather-times`.
```

Comment:
```suggestion
For example, here's a formatter that will remove common stop words from the slug. If the title is entered as `The weather and the times`, this will produce a slug of `weather-times`.
```